### ISSUE TITLE
Rename command arguments

### ISSUE THREAD

**Issue by theiskaa** [anonistas/notya#54]
Currently rename could take only base note's title from arguments, and not the second title - new-title of note.

It would be absolutely good if new-title could be passed as arguments or from flags:
```
> notya rename new_title.txt my_new_note.txt
                   ||            ||
                base file      new title
```

**Comment by theiskaa** [comment]
cc @Melihemin

**Comment by theiskaa** [comment]
Resolved by #57 

Thanks to @Melihemin
